drivers/gpio/gpio_nrf5.c

```diff
@@ -175,6 +183,12 @@ static int gpio_nrf5_config(struct device *dev,
 			pull = GPIO_PULL_DOWN;
 		}
 
+		if (sense == GPIO_SENSE_INVALID) {
+			SYS_LOG_ERR("Invalid parameter for sense,"
+						"setting SENSE disable\n");
+			sense = GPIO_SENSE_DISABLE;
```

Comment:
I thought that you'd rather use an __ASSERT, similarly to what is done a couple of lines above with `ds_low` and 'ds_high`. Such setting of SENSE is simply invalid, and if it is used in an application, it means that there is obviously an error in it. I think the driver should not try to correct it.